### ISSUE TITLE
Possibility to the user only see events related to himself

### ISSUE THREAD

**Issue by samuelsimoes** [samuelsimoes/chrome-basecamp-notifier#9]

**Comment by hugomedeiros** [comment]
Any due date to this? This is an awesome feature!

**Comment by samuelsimoes** [comment]
Hi, @hugomedeiros, glad to see you here. :D

So, it's a bit complicated because until I remember Basecamp hasn't the concept of  task delegation, but even if it has, the API doesn't expose any useful data for this feature.

Other bad news is that Basecamp doesn't supply any kind of environment for development, so in this moment I am without any way to make tests and with the Basecamp 3 arrival we can't create new trial accounts for Basecamp 2. 😔 

TL;DR sadly no due date (Basecamp's fault).
